app/src/main/java/de/westnordost/streetcomplete/quests/QuestModule.java

```diff
@@ -145,6 +146,7 @@
 				new AddPowerPolesMaterial(o),
 				new AddCarWashType(o),
 				new AddBenchBackrest(o),
+				new AddTrafficSignalsButton(o),
```

Comment:
Should not this be prioritized higher (use cases [were given here](https://github.com/westnordost/StreetComplete/issues/574#issuecomment-440344393). I'd just do so that the quest "traffic_signals:sound" is asked _after_ this traffic_signals:button_operated, because it is usually easier to answer whether that thing has a button (you can answer that while standing and waiting at the traffic light), while for the sound you have to wait until you get green.

So IMHO the logical order to answer these quests is:
* button – when I press it
* [wait]
* sound – when I (do not) hear it when it is green

Reply:
"while for the sound you have to wait until you get green", wait, so cases described at https://wiki.openstreetmap.org/wiki/Key:traffic%20signals:sound?uselang=en " there is only the signal to find the pole" and "there is only the signal when walking is allowed" are actually sometimes happening in some regions?

Reply:
On the other hand noone actually uses this values, so it may be probably safely ignored as "someone wanted this tagging scheme and believed that OSM Wiki has magic powers" - see https://taginfo.openstreetmap.org/keys/traffic_signals:sound#values

Reply:
Hmm, did not really thought about that… Maybe yes, I only noticed the "green 'beep'" more often.

Reply:
But I am fairly sure that those traffic lights, where here is "only an accessibility button", i.e. the one that does not influence the traffic light/time to wait, but just enables the sound/accessibility; have no sound by default and only have it when you press that button.